### PR TITLE
Introduce option for image expiration grace period and fix log-level.

### PR DESCRIPTION
**Pull request by Gerrit91**
Adds an option to still sync images even if they have already expired in the metal-api.

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -88,6 +87,8 @@ func init() {
 	rootCmd.Flags().Int("min-images-per-name", 3, "minimum amount of images to keep of an image variant")
 	rootCmd.Flags().Int("max-images-per-name", -1, "maximum amount of images to cache for an image variant, unlimited if less than zero")
 
+	rootCmd.Flags().Int("expiration-grace-period", 0, "the amount of days to still sync images even if they have already expired in the metal-api (defaults to zero)")
```
Comment (majst01) [inline cmd/main.go]: rootCmd.Flags().Uint will make the validation below obsolete

File: pkg/api/config.go [hunk 2]
```diff
@@ -72,5 +75,9 @@ func (c *Config) Validate(fs afero.Fs) error {
 		return fmt.Errorf("minimum images per name must be at least 1")
 	}
 
+	if c.ExpirationGraceDays < 0 {
```
Comment (majst01) [inline pkg/api/config.go]: See comment above about Flags().Uint()

**Review by majst01 — CHANGES_REQUESTED** [review]
One small nit, otherwise LGTM

File: cmd/main.go (hunk 1 above)
Comment (Gerrit91) [reply cmd/main.go]: Done. Thanks for review!

**Review by majst01 — APPROVED** [review]

**Pull request merged**